File: src/main/java/com/baimsg/manual/Own.java
package com.baimsg.manual;

import com.baimsg.Config;
import com.baimsg.thread.DictionaryThreadPoolExecutor;
import com.baimsg.thread.SimpleThread;

import java.io.BufferedReader;
import java.io.File;
import java.io.FileReader;
import java.math.BigInteger;
import java.net.URL;
import java.util.Map;
import java.util.Set;
import java.util.concurrent.ExecutorService;
import java.util.concurrent.Executors;

/**
 * create by baimsg 2021/11/26
 * Email 1469010683@qq.com
 * <p>
 * 实现了手动测试
 **/
public class Own {

    public static void main(String[] args) {
        ExecutorService fixedThreadPool = Executors.newFixedThreadPool(Config.maxThread);

        for (String userName : Config.userNames) {
            fixedThreadPool.execute(new OwnThread(userName));
        }
        fixedThreadPool.shutdown();
    }
}

File: src/main/java/com/baimsg/manual/Own.kt
package com.baimsg.manual

import com.baimsg.common.Config
import kotlin.jvm.JvmStatic
import java.util.concurrent.ExecutorService
import java.util.concurrent.Executors
import com.baimsg.manual.OwnThread

/**
 * create by baimsg 2021/11/26
 * Email 1469010683@qq.com
 *
 *
 * 实现了手动测试
 */
fun main() {
    val fixedThreadPool = Executors.newFixedThreadPool(Config.maxThread)
    for (userName in Config.userNames) {
        fixedThreadPool.execute(OwnThread(userName))
    }
    fixedThreadPool.shutdown()
}
